code/modules/mob/living/carbon/carbon.dm

```diff
@@ -530,7 +530,7 @@
 
 /mob/living/carbon/update_stamina()
 	var/stam = getStaminaLoss()
-	if(stam > DAMAGE_PRECISION && (maxHealth - stam) <= crit_threshold)
+	if(stam > DAMAGE_PRECISION && (initial(maxHealth) - stam) <= crit_threshold)
```

Comment:
should probably be documented this works off initial and not current

Reply:
Applied with https://github.com/tgstation/tgstation/pull/79584/commits/247791f452f3f57f922feacf6aaab663144f2f85